src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -87,6 +122,46 @@ XrdAccPrivs AddPriv(Access_Operation op, XrdAccPrivs privs)
     return static_cast<XrdAccPrivs>(new_privs);
 }
 
+const std::string OpToName(Access_Operation op) {
```

Comment:
Likely this should be made a generic utility in XrdAcc as another implementation of the same thing exists there. Something to put in the todo list.